plugin/trino-hive/src/test/java/io/trino/plugin/hive/TestHiveConnectorSmokeTest.java

```diff
@@ -13,26 +13,34 @@
  */
 package io.trino.plugin.hive;
 
+import com.google.common.collect.ImmutableMap;
+import io.trino.plugin.hive.containers.HiveHadoop;
 import io.trino.testing.BaseConnectorSmokeTest;
 import io.trino.testing.QueryRunner;
 import io.trino.testing.TestingConnectorBehavior;
 import org.testng.annotations.Test;
 
+import static io.trino.plugin.hive.HiveHadoopQueryRunner.createHadoopQueryRunner;
 import static org.assertj.core.api.Assertions.assertThat;
 import static org.assertj.core.api.Assertions.assertThatThrownBy;
 
 // Redundant over TestHiveConnectorTest, but exists to exercise BaseConnectorSmokeTest
 // Some features like views may be supported by Hive only.
+@Test(singleThreaded = true) // Otherwise, CI hangs on the way
 public class TestHiveConnectorSmokeTest
         extends BaseConnectorSmokeTest
 {
     @Override
     protected QueryRunner createQueryRunner()
             throws Exception
     {
-        return HiveQueryRunner.builder()
-                .setInitialTables(REQUIRED_TPCH_TABLES)
```

Comment:
I'm fine hijacking TestHiveConnectorSmokeTest

- remove `Redundant over TestHiveConnectorTest, but exists to exercise BaseConnectorSmokeTest...` comment, since it becomes obsolete
- renme TestHiveConnectorSmokeTest -> TestHiveHdp3ConnectorSmokeTest